src/payments-methods/classic/payment-methods.js

```diff
@@ -1,20 +1,10 @@
-import { decodeEntities } from '@wordpress/html-entities';
-import { getBlocksConfiguration } from '../../utils';
 import { useEffect } from '@wordpress/element';
```

Comment:
I'd suggest to include External/Internal comment to separate dependencies, same as other files

Reply:
Added them, thank you!